NEWS.md

```diff
@@ -33,11 +33,15 @@ Standard library changes
 
 * Added keyword arguments `rtol`, `atol` to `pinv` and `nullspace` ([#29998]).
 
+#### SparseArrays
+
+* `spmatmul` no longer has the obsolete optional keyword argument `sortindices` ([#30372]).
```

Comment:
Isn't this an internal function?

Reply:
I inserted this text on demand of the hint:
https://github.com/JuliaLang/julia/pull/30372#issuecomment-447694306

Reply:
will remove references to `spmatmul`.

Reply:
@KlausC Sorry to create extra work for you.